src/tests/gitpod/gitpodIdManager.test.ts

```diff
@@ -8,23 +8,23 @@ const redhatDir = `${process.cwd()}/.redhat/`;
 
 suite('Test gitpod Id manager', () => {
   setup(() => {  
-      mock({
-        '.redhat': {
-          'anonymousId': 'some-uuid'
-        }
-      });
+      if (fs.existsSync(redhatDir)) {
+        fs.rmSync(redhatDir, { recursive: true, force: true });
+      }
     });
     teardown(() => {
-        mock.restore();
+      if (fs.existsSync(redhatDir)) {
+        fs.rmSync(redhatDir, { recursive: true, force: true });
+      }
     });
     test('Should generate Red Hat UUID from GITPOD_GIT_USER_EMAIL env', async () => {
       env.GITPOD_GIT_USER_EMAIL = 'some.user@company.com';
       console.log(env.GITPOD_GIT_USER_EMAIL);
       const fsIdProvider = new FileSystemIdProvider(redhatDir);
       const gitpod = new GitpodIdProvider(fsIdProvider);
       const id = await gitpod.loadRedHatUUID();
-      const expectedId = '465b7cd6-0f77-5fc8-97ed-7b6342df109f';
```

Comment:
Actually for a given name + namespace, UUID should be constant, independently of the Node version

See https://www.uuidtools.com/api/generate/v5/namespace/44662bc6-c388-4e0e-a652-53bda6f35923/name/some.user@company.com

Reply:
> Actually for a given name + namespace, UUID should be constant, independently of the Node version
> 
> See https://www.uuidtools.com/api/generate/v5/namespace/44662bc6-c388-4e0e-a652-53bda6f35923/name/some.user@company.com

ah my bad, I set it back to the previous vrsion, i was only mixed up in my head with the various changes.
So the probelm was really just related to the mock-fs which was not working anymore